poi.config.js

```diff
@@ -64,6 +64,12 @@ module.exports = {
         files: ['.brandrc', '.brandrc.js'],
         imagesPath
       }
+    },
+    {
```

Comment:
Move opening bracket to the line above. Check previous occurrences and be consistent 💅